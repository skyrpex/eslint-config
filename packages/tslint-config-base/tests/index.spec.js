const { resolve } = require("path");
const { spawnNpmSync } = require("../../../tests/utils");

test("should report TypeScript errors", () => {
    expect(
        spawnNpmSync("tslint", [
            resolve(__dirname, "index.fixture.ts"),
        ]).stdout.toString(),
    ).toMatchSnapshot();
});
