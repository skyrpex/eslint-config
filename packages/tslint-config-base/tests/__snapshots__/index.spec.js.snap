// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`xxx 1`] = `
"
ERROR: packages/tslint-config-base/tests/index.fixture.ts:2:22 - Insert \`;\`
ERROR: packages/tslint-config-base/tests/index.fixture.ts:8:27 - Insert \`,\`
ERROR: packages/tslint-config-base/tests/index.fixture.ts:12:1 - Insert \`路路路路\`
ERROR: packages/tslint-config-base/tests/index.fixture.ts:15:20 - Insert \`;\`

"
`;
